File: Python/p10783.py
kasus = int(input())
for i in range(kasus):
    num1 = int(input())
    num2 = int(input())

    if num1 % 2 == 0:
        num1 += 1

    res = 0

    for j in range(num1, num2 + 1, 2):
        res += j

    print("Case {:d}: {:d}".format(i+1, res))

File: Java/p10783.java
package SOLVED;


import java.util.Scanner;

/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

/**
 *
 * @author DarKnight98
 */
public class p10783 {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        int kasus = input.nextInt();
        
        for (int i = 0; i < kasus; i++) {
            int bawah = input.nextInt();
            int atas = input.nextInt();
            int res = 0;
            if (bawah%2 == 0) {
                bawah ++;
            }
            
            for (int j = bawah; j <= atas; j+=2) {
                res += j;
            }
            
            System.out.printf("Case %d: %d\n", i+1, res);
        }
    }
}
